addons/web_tour/static/src/tour_service/tour_service.js

```diff
@@ -214,14 +193,11 @@ export const tourService = {
             );
 
             if (tourConfig.mode === "auto") {
-                new TourAutomatic(tour).start(pointer, () => {
-                    pointer.stop();
-                    endTour(tour);
-                });
+                new TourAutomatic(tour).start(pointer);
             } else {
                 new TourInteractive(tour).start(pointer, async () => {
                     pointer.stop();
-                    endTour(tour);
+                    browser.console.log("tour succeeded");
```

Comment:
Here we don't call `tourState.clear()` anymore, is this expected?

Reply:
bien vu